nuxeo-web-ui-ftest/test/pages/ui/browser/document_task.js

```diff
@@ -40,4 +40,9 @@ export default class DocumentTask extends BasePage {
     const users = element.elements('nuxeo-user-tag .tag a').value;
     return users.some(user => user.getText() === `${actor} undefined`);
   }
+
+  performAction(name) {
+    this.el.waitForVisible(`.options paper-button[name="${name}"]`);
```

Comment:
The same note as the other PR: probably we should ensure we scrollIntoView before performing the click operation.